nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryQuery.java

```diff
@@ -2096,6 +2096,34 @@ public void testOrderByAndDistinct() throws Exception {
         assertIdSet(dml, file1.getId());
     }
 
+    @Test
+    public void testWildCardsInFulltextSearch() throws Exception {
```

Comment:
Have you run this manually on all supported databases? It's important because T&P won't test this.